README.md

```diff
@@ -1,18 +1,180 @@
-# Workleap.Eventgrid.Emulator
+# Workleap Azure Event Grid Emulator
 
+This Docker image provides a local emulator for **Azure Event Grid**, which can be used for testing and development purposes.
+
+## Prerequisites
+
+Before you can use the EventGridEmulator Docker image, you need to have Docker installed on your machine. You can download Docker from the official website: https://www.docker.com/get-started
 
 ## Getting started
 
-|Package| Download Link                                                                                        | Description                                                                |
-|----|------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
-|    |  |   |
+To use the EventGridEmulator Docker image, you can follow these steps:
 
+1. Create a development certificate ([for more info](https://github.com/FiloSottile/mkcert)):
```

Comment:
It might be worth to explain why we need a certificate - I mean in order to expose the emulator using HTTPS